packages/minimongo/minimongo.js

```diff
@@ -97,6 +97,10 @@ LocalCollection.Cursor = function (collection, selector, options) {
     // stash for fast path
     self._selectorId = selector;
     self.matcher = new Minimongo.Matcher(selector);
+  } else if (LocalCollection._selectorIsIdPerhapsAsObject(selector)) {
```

Comment:
I thought at first that this was wrong because this function also returns true when it's just an ID, but of course that's why it's the else-if here.